modules/wp-seo/wp-seo-front.php

```diff
@@ -38,9 +50,87 @@ function qtranxf_wpseo_webpage_schema( $piece, $context ) {
     add_filter( 'wpseo_breadcrumb_single_link_info', 'qtranxf_wpseo_breadcrumbs_link', 10, 3 );
     function qtranxf_wpseo_breadcrumbs_link( $link_info, $index, $crumbs ) {
         $link_info['text'] = qtranxf_useCurrentLanguageIfNotFoundUseDefaultLanguage( $link_info['text'] );
+        $link_info['url']  = qtranxf_convertURL( $link_info['url'], $lang );
```

Comment:
The `$lang` is not set. This result in using the current language `$q_config['language']`. Is that the intention?
Same below for all filters.

Reply:
Yes, the intention to use current language.

Reply:
Done.